config/README.md

```diff
@@ -69,12 +73,20 @@ global:
 
 - **`mlflow_uri` :** Path where *mlflow* will store all the informations about the runs. By default the path is `./mlruns`.
 
+- **`mlflow_experiment_name` :** Experiment name in *mlflow*.
+
+- **`mlflow_run_name` :** Run name in *mlflow* and preprocessing identification tag.
+
 - **`bucket_name` (Optional) :** Name of the S3 bucket where the data is stored. Leave blank if using local files.
 
 - **`task` :** Task to perform, either segmentation or classification, but classification is no longer supported.
 
 - **`num_gpus` :** Number of **GPUs** that you want to use, `0` will use the **CPU**.
 
+- **`max_used_ram` :** Maximum used **GPUs** memory (MB) in order to use it.
```

Comment:
Is it the max memory available for the run or the max memory already used by another process that won't be taking into account when searching for available GPUs?  
And how does the `max_used_perc` interact with this parameter?  Do both conditions have to be met?

Reply:
max already used. both conditions are checked and have to be met. I reworded in the last commit, is that better?

Reply:
Excellent. Thanks!